automation/test-windows.sh

```diff
@@ -3,7 +3,7 @@
 set -ex
 
 namespace="kubevirt"
-template_name="windows2k12r2"
+template_name=""
```

Comment:
This will break the script if no `TARGET` env was provided.

Reply:
Right, thanks for the review, do you have a preferred default?

Reply:
Let's go with windows2k22 as the current successor.

Please note that this also needs adjustment of the CI jobs in openshift/release. At least the 2k12 lane needs to be removed.

Reply:
Default fixed, and created https://github.com/openshift/release/pull/42370